src/pages/DetailsPage.js

```diff
@@ -66,6 +67,14 @@ const getPhoneNumber = (details) => {
 
 const DetailsPage = (props) => {
     const details = props.personalDetails[props.route.params.login];
+    if (!details) {
+        // Personal details have not loaded yet
+        return (
+            <View style={[styles.flex1, styles.alignItemsCenter, styles.justifyContentCenter]}>
+                <ActivityIndicator color={themeColors.spinner} size="large" />
+            </View>
```

Comment:
NAB, but you could use the `<FullscreenLoadingIndicator />` component here

Reply:
I'll give it a try, thanks!